src/Login.css

```diff
@@ -0,0 +1,66 @@
+//We also need to confirm these, since wireframes and Figma haven't been confirmed yet -- maybe just add a TODO comment:
+//TODO: confirm once mockups provided
+
+.app {
+    font-family: sans-serif;
```

Comment:
Duplicate in font-family:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/126940746/224444846-43682875-9780-49d8-8eb5-ac2eb0bdd1c2.png">